### PR TITLE
Modified tests Feb 07

### PR DESCRIPTION
**Pull request by ghost**
Modified following testcases:
1.TC-DIAG-TH-NW-1.1
2.TC-MC-1.1
3.TC-TM-1.1
4.TC-OCC-1.1
5.TC-TSTAT-2.1
Tested and added auto generated files

Execution log for referenc
[Execution_log.zip](https://github.com/project-chip/connectedhomeip/files/8015316/Execution_log.zip)

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_OCC_1_1.yaml [hunk 1]
```diff
@@ -25,10 +25,11 @@ tests:
 
     #issue #12190 as per spec default value is 3 but expecting 2
     - label: "read the global attribute: ClusterRevision"
+      disabled: true
```
Comment (raju-apple) [inline src/app/tests/suites/certification/Test_TC_OCC_1_1.yaml:28]: Why we adding disabled to these steps now ?

**Comment by woody-apple** [comment]
Fast tracking given this is adding tests.

**Review by woody-apple — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_OCC_1_1.yaml (hunk 1 above)
Comment (manjunathambiger23) [reply src/app/tests/suites/certification/Test_TC_OCC_1_1.yaml:28]: @raju-apple as per spec  default of ClusterRevision is 3 but when we execute it is expecting value to be 2 so we disabled this step.

**Review by andy31415 — APPROVED** [review]

**Pull request merged**